src/views/Scenario/Scenario.js

```diff
@@ -42,7 +56,10 @@ const Scenario = () => {
   const { t } = useTranslation();
 
   // RHF
-  const methods = useForm();
+  const {
+    formState: { isDirty },
+    ...methods
+  } = useForm();
```

Comment:
Doesn't this change remove `formState` from the `methods` object ? Isn't it an issue when we use `<FormProvider {...methods}>` below?